src/stub/parser.rs

```diff
@@ -40,6 +42,7 @@ impl<'a> Parser<'a> {
                 "OUTPUT"    => self.parse_output_comment(&mut stub.commands),
                 "INPUT"     => self.parse_input_comment(&mut stub.commands),
                 "STATEMENT" => stub.statement = self.parse_text_block(),
+                "gameloop"  => return Err(anyhow!("Gameloop keyword is not supported.")),
```

Comment:
CodinGame refers to them as commands so we should too. I'm not sure how to word it well but we should make it clear to a user that they did nothing wrong but there is no bug to report either - ideally even if the user doesn't know anything about CodinGame's stub system.
```suggestion
                "gameloop"  => return Err(anyhow!("Stub generator does not currently support the 'gameloop' command")),
```